app/helpers/application_helper.rb

```diff
@@ -0,0 +1,5 @@
+module ApplicationHelper
+  def is_page_responsive?
```

Comment:
Em Ruby a convenção para esse tipo é método é apenas o `?`, aqui ficou misturado a convenção de JS (`is`) com Ruby. `page_responsive?` seria melhor